lib/commands/backup.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const Command = require('../command');
+const Promise = require('bluebird');
+const createDebug = require('debug');
+const Zip = require('adm-zip');
+const fs = require('fs');
+const path = require('path');
+const cli = require('../');
+
+const debug = createDebug('ghost-cli:backup');
+
+class BackupCommand extends Command {
+    run(argv) {
+        const instance = this.system.getInstance();
+        const config = instance.config.values;
+        const datetime = (new Date()).toJSON().substring(0, 10);
+
+        let exporter;
+        let saveLocation = process.cwd();
+
+        // @todo check if we need to ensure we're in a ghost directory
+
+        if (argv.output) { // @todo add o as an alias
+            saveLocation = path.isAbsolute(argv.output) ? argv.output : path.join(process.cwd(), argv.output);
+            fs.ensureDirSync(saveLocation);
+        }
+
+        saveLocation = path.join(saveLocation, `ghost.backup.${datetime}.zip`)
+
+        instance.checkEnvironment();
+
+        if (instance.running()) {
+            this.ui.log('Ghost is currently running. Backing up might take longer and slow down your site','yellow');
+        }
+
+        const backupData = {
```

Comment:
so honestly this whole part isn't necessary - if you save the `.ghost-cli` file (which you are doing already), then most of this data is redundant.